packages/neuron-wallet/src/services/addresses.ts

```diff
@@ -17,6 +17,9 @@ export interface AddressMetaInfo {
 }
 
 export default class AddressService {
+  // <= 3
+  private static generateCheckDividingLine: number = 3
```

Comment:
Threashold for `Dividing line`?

Reply:
Means if available addresses count <= 3, then generate new addresses.

Reply:
renamed